disnake/client.py

```diff
@@ -265,6 +266,11 @@ class Client:
     asyncio_debug: :class:`bool`
         Whether to enable asyncio debugging when the client starts.
         Defaults to False.
+    session_start_limit: Optional[:class:`SessionStartLimit`]
+        Information about the current session start limit,
+        only set in sharded clients/bots.
```

Comment:
This is a bit of a problem. Session start limit data is also provided when not sharded, and is still important to have. To change, we would need to use the get bot gateway endpoint for non-sharded clients as well.

Reply:
https://github.com/DisnakeDev/disnake/pull/402/commits/200d0ebe3e6abd2cf1fcddea4597287cd072e342